falcon/request_helpers.py

```diff
@@ -238,15 +238,15 @@ def _read(self, size, target):
         return target(size)
 
     def readable(self):
-        """Always returns ``True``."""
+        """Return ``True`` always."""
```

Comment:
Maybe it's just me, but these Return `True`/`False` sound a bit awkward... Could we find an alternative sentence form?

Reply:
I'd love that, but I banged my head against a dictionary (a physical one, not a python one) trying to reword this. It needs to be in imperative mood. I could take out the "always"?

Reply:
Does the imperative need to be first in that `flake8` check?
Maybe just "Always return ``True``" if that is accepted?

Reply:
Hm, could we possibly add a `noqa: D401` for these? :thinking:

Reply:
How about `Return ``True`` in call cases` or `Return ``True`` in every case`?

Reply:
That could also work.

Personally, I vote for simply adding `# noqa: D401` in cases like this.
The docstring _is_ in imperative mood, just the tool is not advanced enough (Deep Learning + [NLP](https://en.wikipedia.org/wiki/Natural_language_processing) patches to the tool anyone? :smiling_imp:).

Reply:
Thanks for the suggestions! Please let me know which you prefer as I don't have an opinion on this (it's pretty minor IMHO).

Reply:
> (Deep Learning + [NLP](https://en.wikipedia.org/wiki/Natural_language_processing) patches to the tool anyone? 😈).

Just a whitelist in this case is way easier

Reply:
@kgriffs any preference on attempting to reword at any cost, or simply adding exceptions where the tool is not smart enough?

I'm fine with either.